src/diffusers/modeling_flax_utils.py

```diff
@@ -436,7 +436,7 @@ def from_pretrained(
             )
             cls._missing_keys = missing_keys
 
-        # Mistmatched keys contains tuples key/shape1/shape2 of weights in the checkpoint that have a shape not
+        # Mismatched keys contains tuples key/shape1/shape2 of weights in the checkpoint that have a shape not
```

Comment:
Actually, it looks like `mismatched_keys` was made obsolete in 73e0bc692c5761e55faff39c80a26d7a3cfc748c. I'll open a new issue to clean it up.